### PR TITLE
Added config for mex 

### PR DESCRIPTION
**Pull request by gsn9**
solves issue #16

### CODE REVIEW

File: SpiceQL/db/mex.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+    "mex":{
+        "ck":{
+            "predicted":{
+                "kernels":"ATNM_P060401000000_01031.BC"
+            },
```
Comment (gsn9) [inline SpiceQL/db/mex.json:6]: I used an actual file name here as the regular expression wan't picking up any file in the path.h file.

File: SpiceQL/db/mex.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        },
+        "ik":{
+            "kernels":"MEX_HRSC_V[0-9]{2}.TI"
+        },
+        "sclk":{
+            "kernels":"MEX_[0-9]{6}_STEP.TSC"
+        },
+        "spk":{
+            "reconstructed":{
+                "kernels":"ORMM__031222180906_00052.BSP"
```
Comment (gsn9) [inline SpiceQL/db/mex.json:28]: Same for both spk reconstructed

File: SpiceQL/db/mex.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+            },
+            "predicted":{
+                "kernels":"ORMM__061001000000_00310.BSP"
+            }
```
Comment (gsn9) [inline SpiceQL/db/mex.json:32]: same here as well

File: SpiceQL/db/mex.json [hunk 4]
```diff
@@ -0,0 +1,35 @@
+{
+    "mex":{
+        "ck":{
+            "predicted":{
+                "kernels":"ATNM_P060401000000_01031.BC"
```
Comment (jessemapel) [inline SpiceQL/db/mex.json:5]: This is going to miss a whole bunch of predict kernels. Needs to be changed to a regex.

File: SpiceQL/db/mex.json [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+            },
+            "reconstructed":{
+                "kernels":[
+                    "ATNM_MEASURED_030602_040101_V03.BC",
```
Comment (jessemapel) [inline SpiceQL/db/mex.json:9]: This will also miss a lot of kernels and needs to be a regex.

File: SpiceQL/db/mex.json [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+                    "ATNM_RECONSTITUTED_[0-9]{5}.BC"
+                ]
+            }
+        },
+        "fk":{
+            "kernels":"MEX_V[0-9]{2}.TF"
+        },
+        "iak":{
+            "kernels":"hrscsrcAddendum[0-9]{3}.ti"
```
Comment (jessemapel) [inline SpiceQL/db/mex.json:18]: This is only the SRC IAK, you also need the HRSC IAK. Should probably split them into separate things that depend on MEX.

File: SpiceQL/db/mex.json (hunk 3 above)
Comment (jessemapel) [reply SpiceQL/db/mex.json:32]: This is a reconstructed kernel, not a predict. Predict SPK are of the form ORMF*

File: SpiceQL/db/mex.json (hunk 2 above)
Comment (jessemapel) [reply SpiceQL/db/mex.json:28]: This will miss all of the other reconstructed kernels, needs to be a regex.

**Pull request merged**